components/notion/sources/new-database/new-database.mjs

```diff
@@ -0,0 +1,46 @@
+import base from "../common/base.mjs";
+import constants from "../common/constants.mjs";
+
+export default {
+  ...base,
+  key: "notion-new-database",
+  name: "New Database",
+  description: "Emit new event when a database is created",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "last",
```

Comment:
Is there a reason to use `dedupe: "last"`?

Reply:
Forgot to remove it. Addressed